terraso_backend/apps/core/landscapes_views.py

```diff
@@ -64,8 +64,8 @@ def to_json(self, landscape):
                     "type": term.type,
                     "value": {
                         "original": term.value_original,
-                        "en": term.value_en,
-                        "es": term.value_es,
+                        "eng": term.value_en,
+                        "spa": term.value_es,
```

Comment:
question: why are we switching to ISO-639-3 here?

Reply:
@paulschreiber This was changed because in the taxonomy terms languages for the orignal value we are using the 3 characters form, so for consistency we decided to change this to use the same format